starter-core/src/test/groovy/io/micronaut/starter/build/maven/MavenBuildTestVerifier.groovy

```diff
@@ -18,8 +18,27 @@ class MavenBuildTestVerifier implements BuildTestVerifier {
         this.language = language
     }
 
+    @Override
+    boolean hasAnnotationProcessor(String groupId, String artifactId) {
+        if (language == Language.GROOVY) {
```

Comment:
replace the method content with : 

` return hasAnnotationProcessor(groupId, artifactId, Scope.ANNOTATION_PROCESSOR);`

Reply:
see https://github.com/micronaut-projects/micronaut-starter/pull/2162/commits/ff4714f83e4e9c55413efc8f53d1dd92f4fe9eb2